### PR TITLE
Slim template

### PR DESCRIPTION
**Pull request by clarksandholtz**
## Summary

Currently, there is only a template that includes several models and a bunch of example code. This template gets you all the necessary dependencies without adding anything extra so you can plug in only the necessary models, dependencies, and source code.

## Changelog

Duplicated the original template and made the following changes:
- Removed the models
- Removed 3rd party dependencies not needed to run a model with PTL
- Simplified `src` folder to contain 1 file called `App.tsx` and replaced it's contents with what used to be in `MyDemos.tsx` modified to make more sense.

## Test Plan

Tested template with `npx react-native init Ejemplo --template file:///Users/clarksandholtz/src/live/react-native-template-ptl` and was able to run the app.

### CODE REVIEW

**Comment by chrisklaiber** [comment]
Thanks, @clarksandholtz! Having a slim template will be nice, especially for experienced PyTorch Live users.

I wonder if we should change the the folder path. The current folder, `react-native-template-ptl`, is very close to the existing  `react-native-template-pytorch-live` and is semantically identical. How about react-native-template-pytorch-live-base` or `-slim` or `-simple`?

Also, is there a file-synchronization linter? For a lot of the boilerplate files, which appears to be almost all of the files, I imagine we want the slim and non-slim template to have identical file contents. It would be awesome if a linter could point to mismatches!

File: react-native-template-ptl/package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "react-native-template-ptl",
+  "version": "0.0.0",
+  "description": "A minimal React Native template for PyTorch Live",
+  "repository": "",
+  "author": "Facebook",
```
Comment (raedle) [inline react-native-template-ptl/package.json:6]:
```suggestion
  "author": "Meta Platforms, Inc.",
```

File: react-native-template-ptl/template.config.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
```
Comment (raedle) [inline react-native-template-ptl/template.config.js:2]: Let's change all license headers to match the new license header for Meta Platforms, Inc.

```suggestion
 * Copyright (c) Meta Platforms, Inc. and affiliates.
```

File: react-native-template-ptl/template/android/app/src/main/java/org/pytorch/live/example/BertQAPacker.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
+/*
```
Comment (raedle) [inline react-native-template-ptl/template/android/app/src/main/java/org/pytorch/live/example/BertQAPacker.java:1]: This file is not needed in the slim template :)

File: react-native-template-ptl/template/package.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
+{
+  "name": "PyTorchLiveExample",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "start-reset-cache": "yarn push-core-package && react-native start --reset-cache",
```
Comment (raedle) [inline react-native-template-ptl/template/package.json:9]: The `push-core-package` script doesn't exist. Should this be removed?

**Review by raedle — CHANGES_REQUESTED** [review]
Overall looks great! A couple of small changes like removing unused Java packer class and updating license headers :)

**Comment by clarksandholtz** [comment]
@raedle thanks for finding those things to change! I removed the BertQA packer file and the extra command in the `start-reset-cache` command. I also changed all the names from Facebook to Meta Platforms.

**Review by raedle — APPROVED** [review]
Looks great! Thanks for the changes!

**Pull request closed without merging**